client/src/planwise/client/scenarios/changeset.cljs

```diff
@@ -44,7 +46,8 @@
      [m/Icon {} (get action-icons "create-provider")]
      [:div {:class-name "icon-list-text"}
       [:p {:class-name "strong"} (if name name (str "Suggestion " ranked))]
-      [:p {:class-name "grey-text"}  (str "Required Capacity: " action-capacity " Coverage: " coverage)]]]]
+      [:p {:class-name "grey-text"} (str "Required Capacity: " action-capacity
+                                         (if (not (nil? coverage)) (str " Coverage: " coverage)))]]]]
```

Comment:
In which cases the coverage might be nil?

Reply:
I'll check why coverage was nil.

Reply:
It is nil when requesting suggestions for an existing provider and it is not nil when requesting suggestions for a new provider. I checked in previous master and behavior is the same.

Reply:
Ok, drop that as a comment in the code and merge. It might come back to haunt us that.